src/Pages/login.jsx

```diff
@@ -0,0 +1,111 @@
+import { event } from "jquery";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+export default () => {
+  const navigate = useNavigate();
+  const [userName, setUserName] = useState(null);
+  const [password, setPassword] = useState(null);
+
+  const [isUsernameActivated, setUsernameActivated] = useState(false);
+  const [isPasswordActivated, setPasswordActivated] = useState(false);
+
+  const [unalert, setUnAlert] = useState(null);
+  const [pwalert, setPwAlert] = useState(null);
+
+  function handleUsername(event){
+    setUserName(event.target.value);
+    
+    checkUsername(userName);
+  
+  }
+
+  function handleUnBlur() {
+    setUsernameActivated(true)
+    checkUsername(userName)
+  }
+    function handlePwBlur() {
+      setPasswordActivated(true);
+      checkPassword(password)
+    }
```

Comment:
We should be mindful of correct indentation in the code we add.
```suggestion

  function handlePwBlur() {
    setPasswordActivated(true);
    checkPassword(password)
  }
```
It'll keep the code clean and more readable as we progress with a component.

Reply:
Fixed e97eae75b859e2f1f70c09d6abe315d63c384f20